src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -53,6 +53,7 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
     } else {
       try {
         await requestCRReview({ userIds: reviewerIds });
+        toast.success('Request Review successful');
```

Comment:
Nice. One change, I think we should change this to say: "Successfully Requested Review From: Thomas Emrax, Joe Schmoe, etc." where you use the userIds to get the users' names from the list of all users in this file